homeassistant/components/roborock/switch.py

```diff
@@ -165,11 +165,10 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         )
 
     @property
-    def is_on(self) -> bool | None:
+    def is_on(self) -> bool:
         """Return True if entity is on."""
-        return (
+        return bool(
             self.get_cache(self.entity_description.cache_key).value.get(
```

Comment:
This can return `None`? Which in that becomes `False`, which isn't correct?

Reply:
I now return none if the status is None, does that solve your question?